house/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+
+
+class Country(models.Model):
+    name = models.CharField(max_length=30)
```

Comment:
Same as City class

Reply:
a Country could be a TextChoice possibly but I think we could leave it a Class for making it a dynamic list because we can easily use a REST Api like https://restcountries.com/ to make the City and Country a dynamic list